apps/portal/i18n/locales/it.json

```
@@ -0,0 +1,779 @@
+    "manageTeams": "Gestisci Team",
+    "createTeam": "Crea Team",
+    "leaveTeam": "Lascia il team",
+    "noTeams": "Non ci sono team per questa area di lavoro...",
+    "inviteteammatesdescription": "Crea un team e aggiungi membri.",
+    "teamOwner": "Proprietario del Team",
+    "newWorkspaceName": "Nuovo Nome dell'Area di Lavoro",
+    "linkToWorkspace": "Condividi link con il team",
+    "description": "Collaborazione semplificata con un'area di lavoro per condividere immagini e video istantaneamente.",
+    "addWorkspace": "Aggiungi Area di Lavoro",
+    "enterAWorkspaceName": "Inserisci un nome per l'area di lavoro",
+    "renameWorkspace": "Rinomina Area di Lavoro",
```

Biome (1.9.4)

[error] The key renameWorkspace was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)